src/mpbuild/build.py

```diff
@@ -67,7 +61,7 @@ def build_board(
         f"{build_container} "
         f'bash -c "'
         f"git config --global --add safe.directory '*' 2> /dev/null;"
-        f'make -C mpy-cross && make -C ports/{port} submodules all BOARD={board} {args}"'
+        f'make -C mpy-cross && make -C ports/{port} submodules all BOARD={board} {variant} {args}"'
```

Comment:
I don't like that if `variant` and `args` are not specified we get an additional two spaces - but the workarounds are annoying!

Reply:
Just try `variant = f"BOARD_VARIANT={variant}" if variant else r"\b"` ;-)

Reply:
You know, I had the same silly thought too 😝

Reply:
OK, the fix wasn't too ugly...